src/test/javascript/spec/component/tutorial-groups/tutorial-groups-management/tutorial-group-sessions/tutorial-group-sessions-management/tutorial-group-session-row-buttons.component.spec.ts

```diff
@@ -1,4 +1,4 @@
-import { ComponentFixture, TestBed, fakeAsync } from '@angular/core/testing';
+import { ComponentFixture, TestBed, fakeAsync, tick } from '@angular/core/testing';
```

Comment:
Unused import

Reply:
Fixed in de7d568fe3e1c97706f11fc94e414379e6d1bd47